FakeDatabase/campaign-data.json

```diff
@@ -0,0 +1,296 @@
+			]
+		},
+		{
+			"campaign": "a12d3",
+			"name": "Not so Flawless",
+			"key": "Cr4ck",
+			"description": "Cordelia and the Masquerade have a tenuous relationship at best... meaning she's breached it three times already.  The old Prince had a soft spot for her and was willing to overlook her transgressions.  The new one?  Not so much.  She either gets her act together or she'll be catching a sunrise in the near future.",
+			"createdBy": "Bob",
+			"currentGoals": [
+				"Curry favor with the new Prince and her court",
+				"Win over Dorian"
+			]
+		},
+		{
+			"campaign": "a12d3",
+			"name": "Meet and Greet",
+			"key": "PILOT",
+			"description": "The new Prince is hosting a party to celebrate her new title.  Cordelia and Lyle are 'unexpectedly' invited.",
+			"createdBy": "Tom",
+			"currentGoals": []
+		},
+		{
+			"campaign": "a12d3",
+			"name": "Sharp Threads",
+			"key": "sHaRp",
+			"description": "Hell hath no fury like a Toreador scorned.  Internationally renowned designer Clarissa has her vengeful eyes set on a certain pair of miscreants who stole from her.",
+			"createdBy": "Tom",
+			"currentGoals": ["Ask Clarissa what can be done to appease her wrath."]
+		},
+		{
+			"campaign": "a12d3",
+			"name": "A Date to Die for",
+			"key": "flert",
+			"description": "Cordelia isn't too keen on the new Prince.  The Sherif, Dorian, on the other hand, may make for a valuable ally.  Perhaps with a little sweet talk, he could even be turned against the Prince.",
+			"createdBy": "Bob",
+			"currentGoals": ["Wait for Dorian to schedule a date."]
+		},
+		{
+			"campaign": "a12d3",
+			"name": "Gone but not Forgotten",
+			"key": "aliens",
+			"description": "About 2 months ago, 40 prominent Kindred disappeared from the city without leaving a trace.  At this point, many have given up hope of their safe return...  And fear ligers throughout the streets that it may happen again.",
+			"createdBy": "Tom",
+			"currentGoals": [
+				"Figure out who the victims were",
+				"Figure out what exactly happened"
+			]
+		},
+		{
+			"campaign": "a12d3",
+			"name": "Leaving Loose Ends",
+			"key": "L00s3",
+			"description": "Cordelia was already on thing ice for masquerade violations.  Now, she's led one more human to questioning whether the supernatural exist.  The Prince expects her to deal with it quietly.",
+			"createdBy": "Tom",
+			"currentGoals": ["Find a nonviolent way to deal with the warehouse manager."]
+		}
+	],
+	"milestones": [
+		{
+			"description": "Cordelia forces Lyle to go shopping for a new party dress with her at a hot new outlet owned by a famous Toreador.",
+			"occurred": "2024-07-21T13:32:16.581Z",
+			"session": "session1",
+			"addedBy": "Tom",
+			"quests": ["PILOT"]
+		},
+		{
+			"description": "Cordelia steals a dress.  The store owner is not amused.",
+			"occurred": "2024-07-21T13:37:14.578Z",
+			"session": "session1",
+			"step": 2,
+			"addedBy": "Tom",
+			"quests": ["PILOT", "sHaRp"]
```

Comment:
Ignore - Readme answered my question

Interesting that a milestone can correspond to multiple quests.  My first guess would be to picture a milestone as living on a specific quest.  i.e. if you want to write a milestone, then you first open a quest, then you click a button to add a milestone to it.

I can happily chalk this up to my hazy view of the product's end state, though.

Reply:
No worries, I started with the same understanding of "one milestone belongs to one quest"... but once I started thinking through what happened in my actual campaign, I realized how often one character action resolved or advanced multiple quests.  I didn't like the idea of having copies of the same data scattered across quests.  I thought that would make it harder to manage and update.